tests/test_correctness.py

```diff
@@ -3,18 +3,21 @@
 from umbral import pre, keys
 from umbral.point import Point
 from umbral.fragments import CorrectnessProof
+from umbral.signing import Signer
 from .conftest import parameters
 
 
 def test_correctness_proof_serialization():
-    priv_key_alice = keys.UmbralPrivateKey.gen_key()
-    pub_key_alice = priv_key_alice.get_pubkey()
+    priv_key_alice_deleg = keys.UmbralPrivateKey.gen_key()
+    pub_key_alice_deleg = priv_key_alice_deleg.get_pubkey()
```

Comment:
Yes, in pyUmbral, we don't have "Policies" per se.

Also, remember that we use `pubkey` and `privkey` with no spaces.

Reply:
Well, we don't currently use `pubkey` and `privkey` with no space throughout most of pyUmbral - I was keeping the convention that was there (instead of refactoring the entire codebase).  I can.  This is another example of needed polish.

And, as I have said before, although we don't *model* a policy in pyUmbral, we still unquestionably *have one*.  We don't just tell alice is scatter the kfrags to the wind - she has to keep them together and distribute them according to... what?  Her policy preference, no?

I mean, I don't care that much.  But I do think that `pubkey_alice_sig` *and* `pubkey_alice` both being in many of the tests is confusing, compared to either of the alternatives.